packages/core/src/components/tag-input/tag-input.md

```diff
@@ -1,9 +1,9 @@
-@# TagInput
+@# Tag inputs
 
-`TagInput` renders [`Tag`](#core/components/tag)s inside an input, followed by an actual text input. The container is merely styled to look like a Blueprint input; the actual editable element appears after the last tag. Clicking anywhere on the container will focus the text input for seamless interaction.
+Tag inputs render [`Tag`](#core/components/tag)s inside an input, followed by an actual text input. The container is merely styled to look like a Blueprint input; the actual editable element appears after the last tag. Clicking anywhere on the container will focus the text input for seamless interaction.
```

Comment:
hmm I preferred the `TagInput` usage to clarify that it's one thing (not two words) and that there's only a React API for it.